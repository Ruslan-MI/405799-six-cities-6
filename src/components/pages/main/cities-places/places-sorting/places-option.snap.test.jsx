import React from "react";
import {
  render,
} from "@testing-library/react";
import PlacesOption from "./places-option";
import {
  StoreNameSpace,
} from "../../../../../const";
import Test, {
  mockMainPageInitialState,
} from "../../../../../mocks-for-test";

it(`Places option should render correctly`, () => {
  const onClick = jest.fn();

  const dataForStore = {
    [StoreNameSpace.MAIN_PAGE]: mockMainPageInitialState,
  };

  const {
    container,
  } = render(<Test dataForStore={dataForStore}>
    <PlacesOption sortType={mockMainPageInitialState.currentSortType} onClick={onClick} />
  </Test>);

  expect(container).toMatchSnapshot();
});
