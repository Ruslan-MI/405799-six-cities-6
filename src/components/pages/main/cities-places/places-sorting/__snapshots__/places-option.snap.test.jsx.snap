// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Places option should render correctly 1`] = `
<div>
  <li
    class="places__option places__option--active"
    tabindex="0"
  >
    Popular
  </li>
</div>
`;
